### ISSUE TITLE
[WebScrobbler] Can't use slugs

### ISSUE THREAD

**Issue by Magikh42** [FoxxMD/multi-scrobbler#137]
**Describe the bug**
Hello, I tried to use Multi-Scrobbler in order to scrobble from Spotify and WebScrobbler
It seems WebScrobbler works only if there is no slug in the config file

**To Reproduce**
- Setup WebScrobbler source with a slug
- Setup Maloja (or other) client
- Restart (I use Docker so `docker restart multi-scrobler`)
- Set the webhook url in WebScrobbler (`https://scrobbler.mydomain.tld/api/webscrobbler/magikh`)

**Expected behavior**
It shoulds works even when using a slug

**Logs**
Nothing releated to Web Scrobbler appears in the logs

**Config file** (config.json)
```
{
	"sources": [
		{
			"name": "lastfm-magikh42",
			"type": "lastfm",
			"enable": false,
			"clients": [
				"maloja-magikh42"
			],
			"data": {
				"apiKey": "REDACTED",
				"secret": "REDACTED",
				"redirectUri": "http://scrobbler.mydomain.tld/lastfm/callback"
			}
		},
		{
			"name": "webscrobbler-magikh",
			"type": "webscrobbler",
			"enable": true,
			"clients": [
				"maloja-magikh"
			],
			"data": {
				"slug": "magikh", // If I remove this line and edit the webhook in the Web Scrobbler addon settings it works
				"whitelist": ["Invidious"], // I also tried with no blacklist nor whitelist 
				"blacklist": []
			}
		},
		{
			"name": "spotify",
			"type": "spotify",
			"enable": true,
			"clients": [
				"maloja-magikh42"
			],
			"data": {
				"clientId": "REDACTED",
				"clientSecret": "REDACTED",
				"redirectUri": "http://scrobbler.mydomain.tld/spotify/callback",
				"interval": 20
			}
		}
	],
	"clients": [
		{
			"name": "maloja-magikh",
			"type": "maloja",
			"enable": true,
			"data": {
				"url": "https://maloja.mydomain.tld",
				"apiKey": "REDACTED"
			},
			"enable": true
		}
	]
}
```

**Versions (please complete the following information):**
Provide version information for any related sources/clients.

- multi-scrobbler: 0.6.5 on Docker
- maloja: 3.2.2 on Docker
- WebScrobbler: 3.5.0 on Firefox

Also, I use Multi-Scrobbler behind a reverse proxy (`scrobbler.mydomain.tld` -> 'localhost:9078')

**Comment by Magikh42** [comment]
EDIT: I have this line in the log when I try to scrobble using a slug
```
2024-02-14T04:45:36+00:00 debug   : [API] Server received POST request from ::ffff:172.17.0.1 (REDACTED ipv6) (UA: Mozilla/5.0 (Windows NT 10.0; rv:102.0) Gecko/20100101 Firefox/102.0) to unknown route: /
```

**Comment by FoxxMD** [comment]
Thanks for the detailed logs. Please try out `foxxmd/multi-scrobbler:develop` docker image and see if this issue is resolved for you

**Comment by FoxxMD** [comment]
Hi @Magikh42 have you had a change to verify this fix worked for you?

**Comment by Magikh42** [comment]
Hi, I was away from home but I'll soon check if using the development version solve the issue

**Comment by FoxxMD** [comment]
@Magikh42 did you get to test this? I'm ready to cut a release but want to confirm this is fixed.

**Comment by Magikh42** [comment]
I tried with the develop branch, the slug feature seems to be working as intended
Thanks for the fix !

**Comment by FoxxMD** [comment]
Fix is included in [0.7.0](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.7.0)
